src/test/debuginfo/pretty-std.rs

```diff
@@ -40,14 +39,15 @@
 // gdbr-check:$6 = core::option::Option::None
 
 // gdb-command: print os_string
-// gdb-check:$7 = "IAMA OS string 😃"
+// gdb-check:$7 = "IAMA OS string"
```

Comment:
I don't think it's essential. In my terminal emulator this prints as the emoji, but on CI it seems to print escaped to octals. I did not find a quick way to make print the same everywhere, so I figured that it's better to have a test for the simple case instead of none at all.